test/python/quantum_info/test_synthesis.py

```diff
@@ -549,6 +541,10 @@ def test_euler_basis_selection(self, euler_bases, kak_gates, seed):
             unitary = random_unitary(4, seed=seed)
             self.check_exact_decomposition(unitary.data, decomposer)
 
+            a = 2 * np.pi * np.random.random()
+            unitary = Ud(a, 0, 0)
+            self.check_exact_decomposition(unitary, decomposer)
```

Comment:
This doesn't seem to belong here (this is checking the euler basis selection, and its best to check only one thing at a time for unit tests). I have some more complete tests that I'll add after this PR lands, I think it's ok to just remove this.